src/main/resources/org/padaiyal/utilities/commandline/CommandLineUtility.properties

```diff
@@ -0,0 +1,18 @@
+CommandLineUtility.linux.which=which
```

Comment:
Remove unnecessary 'suppress inspections'.

Reply:
It's needed for IntelliJ or any other IDE to not generate a warning.